client-src/overlay.js

```diff
@@ -131,7 +131,8 @@ function show(messages, type) {
       const entryElement = document.createElement("div");
       const typeElement = document.createElement("span");
 
-      typeElement.innerText = type === "warnings" ? "Warning:" : "Error:";
+      typeElement.innerText =
+        (type === "warnings" ? "Warning:" : "Error:") + message.file || "";
```

Comment:
I think `message.file` is `undefined` for webpack 4. So we should only add it when it's defined.

Reply:
I have add comment for it.  file only work for typescript